front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx

```diff
@@ -50,6 +55,7 @@ type ManchetteWithSpaceTimeChartProps = {
 export const MANCHETTE_WITH_SPACE_TIME_CHART_DEFAULT_HEIGHT = 561;
 const BOTTOM_TOOLBAR_HEIGHT = 40;
 const SPACE_TIME_CHART_DIFF_HEIGHT = 8;
+const DEFAULT_ZOOM_MS_PER_PX = 7500;
```

Comment:
Seems like this const is imported from osrd-ui above?

Reply:
yes, was just waiting for the ui release